docs/content/developers/building-contributing.md

```diff
@@ -67,21 +67,28 @@ brew link --force ddev
 
 ### Installing a Downloaded Binary in the `$PATH`
 
-Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `~/bin` in `$PATH`, you can create the folder and add it to your path in `~/.bashrc` with these commands:
+Normally, you can put any executable in your path, and it takes precedence, so you don't need to remove or disable an already installed DDEV instance, which we will use here. This example uses `~/bin`. `echo $PATH` and `which ddev` are valuable commands for debugging. Since not every distro has `~/bin` in `$PATH`, you can create the folder and add it to your path.
 
+Add `~/bin` to your path, by adding this in your `~/.bashrc` file:
 ```
-mkdir -p ~/bin
 export PATH="~/bin:$PATH"
 ```
+Create the `~/bin` folder:
+```
+mkdir ~/bin
+```
+Close and reopen your terminal, and verify that it worked with `echo $PATH`.
 
-Next, unzip the ZIP file you downloaded, make it executable, and move it to a folder in your path. Check with `echo $PATH`:
+Next, download the executable file inside the `~/bin` folder, unzip it, and make it executable:
```

Comment:
We download a ZIP archive, not a file.
It's better to add a real example with `wget` and `unzip`, because nobody probably uses that. (I will add it.)

Reply:
Sure, I am open for corrections and suggestions.